templates/form_themes/adherent_email_subscription_theme.html.twig

```diff
@@ -0,0 +1,18 @@
+{% block adherent_email_subscription_label %}
+    <h2>Notifications par emails activées</h2>
+    <button type="button" class="btn--green click-all-checkbox" data-target="{{ form.emails_subscriptions.vars.id }}">Activer</button>
```

Comment:
why this stuff is in form template? IMO it can be moved in template of form directly

Reply:
You mean in widget ? These buttons are parts of this form, and in mock-up it's rendered on right side of label form.

Reply:
Agree with @ottaviano 

It's overkill, form theme should not have any content.

Form_theme are a way of decoupling form rendering from the model. That way we can easily change the form rendering across all the app.

So we can copy/paste this content within the template where `email_subscription` field is rendered.

Just include a template or whatever other solution that is correct if you want to reuse that anywhere else.